packages/perseus-editor/src/components/coordinate-pair-input.tsx

```diff
@@ -17,7 +17,18 @@ type Props = {
 };
 
 const CoordinatePairInput = (props: Props) => {
-    const {coord, labels, error, range, onChange} = props;
+    const {
+        coord,
+        labels = ["x coord", "y coord"],
+        error,
+        range,
+        onChange,
+    } = props;
+
+    const [xRangeError, setXRangeError] = React.useState<string | null>(null);
+    const [yRangeError, setYRangeError] = React.useState<string | null>(null);
```

Comment:
I did separate `xRangeError` and `yRangeError` instead of `rangeError: [string, string]`, because using an array was causing the component to rerender infinitely from the `useEffect`. I could have kept the array by doing a deep equality check, but it seemed excessive for an array of 2 elements.